examples/CodeGen/deploy/gaudi.md

```diff
@@ -0,0 +1,375 @@
+# Single node on-prem deployment with TGI on Gaudi AI Accelerator
+
+This deployment section covers single-node on-prem deployment of the CodeGen
+example with OPEA comps to deploy using the TGI service. We will be showcasing how
+to build an e2e CodeGen solution with the CodeLlama-7b-hf model,
+deployed on Intel® Tiber™ AI Cloud (ITAC). To quickly learn about OPEA in just 5 minutes and set up the required hardware and software, please follow the instructions in the
+[Getting Started](https://opea-project.github.io/latest/getting-started/README.html) section. If you do
+not have an ITAC instance or the hardware is not supported in the ITAC yet, you can still run this on-prem.
+
+## Overview
+
+The CodeGen use case uses a single microservice called LLM. In this tutorial, we 
+will walk through the steps on how on enable it from OPEA GenAIComps to deploy on 
```

Comment:
`how on enable it` -> `how to enable it`

Reply:
fixed